### PR TITLE
Chore/e2e upload

### PR DESCRIPTION
**Pull request by kendelljoseph**
## Description
- Updates the e2e tests for uploads
- Corrects bug with upload cropping

## Type of change

- [x] Chore (non-breaking change which does not add functionality)
- [x] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: test/uploads/e2e.spec.ts [hunk 1]
```diff
@@ -193,12 +194,6 @@ describe('uploads', () => {
     await expect(listDrawer).toBeVisible()
     await wait(200) // list is loading
 
-    // ensure the only card is the audio file
```
Comment (kendelljoseph) [inline test/uploads/e2e.spec.ts:196]: This check was removed because the seeded config has changed.

File: test/uploads/e2e.spec.ts [hunk 2]
```diff
@@ -76,7 +76,7 @@ describe('uploads', () => {
     await wait(110)
     const field = page.locator('.cell-image')
 
-    await expect(field).toContainText('image.png')
+    await expect(field).toContainText('<No Image>')
```
Comment (denolfe) [inline test/uploads/e2e.spec.ts:79]: Why was this one changed? Seems like `<No Image>` would contradict the name of the test.

File: test/uploads/e2e.spec.ts [hunk 3]
```diff
@@ -288,18 +286,17 @@ describe('uploads', () => {
         await expect(page.locator('.edit-upload__input input[name="X %"]')).toHaveValue(`${focalX}`)
         await expect(page.locator('.edit-upload__input input[name="Y %"]')).toHaveValue(`${focalY}`)
 
+        // apply crop
         await page.locator('button:has-text("Apply Changes")').click()
         await page.waitForSelector('button#action-save')
         await page.locator('button#action-save').click()
+        await wait(1000) // Wait for the save
```
Comment (denolfe) [inline test/uploads/e2e.spec.ts:293]: Is there a toast for this operation that we can listen in on? If so, we can use something like 

```
await expect(page.locator('.Toastify__toast--success')).toHaveText(
  'Deleted 3 Posts successfully.',
)
```

Comment (kendelljoseph) [reply test/uploads/e2e.spec.ts:293]: This worked - adding this check.
```javascript
await expect(page.locator('.Toastify')).toContainText('successfully')
```

**Pull request closed without merging**